modules/vaos/app/serializers/eps/draft_appointment_serializer.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Eps
```

Comment:
Assuming this serializer change is for just adding rspecs & final serializer changes come from [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/20337)

Reply:
yes, that's correct